src/main/resources/labels.json

```diff
@@ -11758,6 +11758,14 @@
         "multilangLabel": { }
     },
     {
+        "uri": "http://purl.org/ontology/bibo/Image",
+        "label": "Bild",
+        "name": "image",
```

Comment:
As this is a class it should not be written with 1st letter upper case. Generally, we only need `"uri"` and `"name"` information for the classes, see also #59.